Eastern-Research-Group/mywaterway
# Feature/411 monitoring lat lon

## Related Issues:
* [HMW-411](https://jira.epa.gov/secure/RapidBoard.jspa?rapidView=864&view=detail&selectedIssue=HMW-411)

## Main Changes:
* Added location latitude and longitude to the USGS streamgage accordion panels, the WQP monitoring locations panels, and the individual monitoring report pages. Since most values come in with 5 points of precision, I used that as the value, but we can add more precision if necessary.
* Fixed an issue with the query used to fetch surrounding streamgages by extent. I'm not sure what changed, or why the previous query worked before, so we'll need to keep an eye on it.

## Steps To Test:
1. Go the Washington DC's Monitoring tab: http://localhost:3000/community/dc/monitoring.
2. Open a streamgage info panel and confirm that _Latitude/Longitude_ is displayed below the _Water Type_ row.
3. In the neighboring "Past Water Conditions" tab, open a monitoring location panel and confirm the same row is present.
4. Open the corresponding **Monitoring Report** page, and confirm that the same values are shown in the info box at the top of the page.
5. Confirm that the surrounding streamgages layer populates correctly.

## TODO:
- [ ] Showing the latitude and longitude for dischargers was briefly discussed at the last meeting, but I'm not sure if a decision was made.
